src/lib/server/index.ts

```diff
@@ -12,12 +12,145 @@ let db: DB | null = null;
 export async function getDb(): Promise<DB> {
     if (!db) {
         db = await createDb(POSTGRES_CONNECTION_STRING);
+        init_db(db);
```

Comment:
We likely want that to be protected behind an environment variable like `SEED` or something.

Reply:
Or perhaps it can be protected behind the deployment environment variable. So only run in dev or something.